### ISSUE TITLE
T10.16. Чи може клас зовсім не мати конструкторів?

### ISSUE THREAD

**Issue by WAG13** [kzhereb/knu-is-ooop2019#16]

**Comment by WAG13** [comment]
За замовчуванням клас уже має конструктор, тому прописувати його не обов'язково.

**Comment by i1red** [comment]
Ні, якщо ви не описали жодного конструктора, то буде викликатися конструктор, створений компілятором за замовчуванням

**Comment by solovdaria** [comment]
Ні, не може. Якщо в класі не буде явно прописаного конструктора, то компілятор створить конструктор за замовчуванням, визначений неявним способом.

**Comment by Velly69** [comment]
Ні, не може, тому що якщо не існує визначених явним способом конструкторів в класі, то компілятор використовує конструктор за замовчуванням, визначений неявним способом та зазвичай є конструктором, який не має аргументів.

**Comment by AntonHladkiy** [comment]
Ні , якщо конструкторів немає то він створюється компілятором за замовчуванням.

**Comment by RostyslavMV** [comment]
Навіть якщо в деяких сучасних компілятор в різних мовах буває можливість не писати конструктор за замовчуванням, то він створиться автоматично, але цього краще уникати.

**Comment by ilyakorenevskyi** [comment]
Конструктор можна не написати (але не бажано),  але компілятори створюють його самі за замовчуванням
